_data/changelogs/about-monthly-calls.yml

```
@@ -2,6 +2,10 @@ title: Monthly calls
 type: documentation
 changelogURL:
 items:
+ - date: 2024-10-00
+    summary: Added August 2024 monthly call video.
+    githubPr: 2846
```

@finekatie Looks like this just is missing a space to get it to build. I also added a few tweaks to emphasize this update just adds the video and added the correct PR number. 

Related: I am not sure that we actually _need_ a changelog for adding just the video, but I am leaning towards its inclusion here just to err on the side of clarity. But I think it is something we can discuss if desired.

```suggestion
  - date: 2024-10-23
    summary: Added video for August 2024 monthly call.
    githubPr: 2873
```

---

Works for me! Thank you for looking!